client/my-sites/plan-features-comparison/index.jsx

```diff
@@ -364,17 +365,18 @@ export default connect(
 				const planConstantObj = applyTestFiltersToPlansList( plan, undefined );
 				const planProductId = planConstantObj.getProductId();
 				const planObject = getPlan( state, planProductId );
-				const showMonthly = ! isMonthly( plan );
+				const isMonthlyPlan = isMonthly( plan );
+				const showMonthly = ! isMonthlyPlan;
 				const availableForPurchase = true;
 				const relatedMonthlyPlan = showMonthly
 					? getPlanBySlug( state, getMonthlyPlanByYearly( plan ) )
 					: null;
 
-				let popular;
-				if ( monthlyDisabled && planObject?.product_name_short === 'Business' ) {
-					popular = true;
-				} else if ( monthlyDisabled ) {
-					popular = false;
+				let popular = false;
+				if ( monthlyDisabled && isMonthlyPlan ) {
+					if ( planObject?.product_name_short === 'Business' ) {
+						popular = plan;
```

Comment:
This is the only condition that will assign a string value, `business-bundle-monthly`, to `popular` instead of a boolean flag. Is it intentional?

Reply:
@southp No, it isn't. I think it should be true. Let me fix it quickly. 😄